### DISCUSSION TITLE
Learning Individual course

### DISCUSSION THREAD

**Discussion by odeddror** [TabularEditor/TabularEditor3#1011] [Learn]
Hi there,

I'm just wondering why I can't brows any course I want without finish all course before the course I wanted to watch.
I the Learn page I wanted to see course 7 but it forces me to start from course 1.

Thanks,
Oded Dror

**Comment by otykier** [comment]
Hi Oded,
I am not able to reproduce this behavior. When I go to [tabulareditor.com/learn](https://tabulareditor.com/learn) and click on Course 7, for example, I get taken to this page, even if I have not completed the previous courses on this account.

![image](https://github.com/TabularEditor/TabularEditor3/assets/8976200/d0e72117-a3a7-481e-873d-7d7feefbe468)

Could you share a screenshot or small recording that displays the issue in your case? Thanks.

**Reply by odeddror** [reply]
I same like mine but how do you browse this course? 
![Course8](https://github.com/TabularEditor/TabularEditor3/assets/9030027/f106cf72-50a3-4218-ba3f-c2514980cc1b)

There are no clickable option that take you to this course?

Thanks,
Oded Dror

**Reply by otykier** [reply]
You should be able to click anywhere in the "Section 1: Managing Incremental Refresh"-box to start the course.

![image](https://github.com/TabularEditor/TabularEditor3/assets/8976200/a7b6c591-e25b-4a55-bf22-d00e227edb71)

**Reply by otykier** [reply]
And just to clarify, we have 9 **courses** available (1-9). Each course contains several **sections** and typically one or more **business cases**. You choose a **course** on the [tabulareditor.com/learn](https://tabulareditor.com/learn)-page, and then you are redirected to elearning.easygenerator.com, from where you can start the **sections** of that course.

So the overall structure of the training is something like:

1. **Course 1**: Introduction to Tabular Editor
  1.1. **Section 1**: What is Tabular Editor?
  1.2. **Section 2**: Example use cases
  1.3. **Section 3**: Getting set up
  1.4. **Section 4**: File Formats
  1.5. **Businses Case**: Introduction to Tabular Editor
2. **Course 2**: Open & Browse Your Model
  2.1. **Section 1**: Connecting to a Model
  2.2. **Section 2**: User Interface & Default Windows
  2.3. **Section 3**: Configuration
  2.4. **Section 4**: Customization & Accessibility
  2.5. **Section 5**: Mastering the TOM Explorer
  2.6. **Section 6**: TOM Properties
  2.7. **Section 7**: Diagram View
  2.8. **Business Case**: Open & Browse Your Model
3. **Course 3**: ...
   3.1. ...
   3.2. ...
4. ...

etc.

**Reply by odeddror** [reply]
Daniel,

Thanks it works

Thanks,
Oded Dror
